index.js

```diff
@@ -67,6 +67,44 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(e) {
+    toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('button.like-button')
+const raffyLikes = document.querySelector('p.likes')
+
+likeButton.addEventListener('click', function() {
+    traveler.likes += 1
+    raffyLikes.innerHTML = `${traveler.likes} Likes`
+})
+
+
 /***** Deliverable 3 *****/
+
+const newSightingForm = document.querySelector('form#new-animal-sighting-form')
+
+newSightingForm.addEventListener('submit', function(event) {
+    event.preventDefault()
+
+    const speciesInput = event.target.species.value
+    const videoInput = event.target.link.value
+    const photoInput = event.target.photo.value
+    const descriptionInput = event.target.description.value
+
+    const lastIndex = traveler.animalSightings.length - 1
+    const lastId = traveler.animalSightings[lastIndex].id
+  
+    const newSightingObj = {
```

Comment:
Good! You can also be fancy with ES6 syntax if you set both the keys and values to be exactly what they are on the form like...

```javascript
const newData = {
        species,
        videoLink,
        photo,
        description
    }
```
but not required and I think your approach is clean and easy to read.

What if we wanted to reset the form after submitting? Maybe there's a function that lives on the `event.target` object that lets us do that... 🤔